src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+import { Dialog } from "@material-ui/core";
+
+interface ModalProps {
+  title: string;
+  image: string;
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    paper: {
+      width: 350,
+      height:70,
+      boxShadow: theme.shadows[5],
+      padding: theme.spacing(2, 4, 3),
```

Comment:
Could we make this properly centered by making the bottom padding the same as the top padding? 
```suggestion
      padding: theme.spacing(2, 4),
```